sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/metrics/QueryMetricsTextWriter.java

```diff
@@ -64,7 +64,7 @@
     private static final String ACTIVITY_ID_HEADER = "Activity Id";
 
     // Constants for Scheduling Metrics Table
-    private static final String PARTITION_ID_HEADER = "Partition Id";
+    private static final String PARTITION_ID_HEADER = "Partition Feed Range";
```

Comment:
we should use a different naming or add a new property

PARTITION_FEED_RANGE_HEADER

Reply:
Done